docs/SIL.rst

```diff
@@ -8006,6 +8006,25 @@ the basic block argument will be the operand of the ``throw``.
 
 A function must not contain more than one ``throw`` instruction.
 
+throw_addr
+``````````
+::
+
+  sil-terminator ::= 'throw_addr'
+
+  throw_addr
+  // indirect error result must be initialized at this point
+
+Exits the current function and returns control to the calling
+function. The current function must have an ndirect error result,
+and so the function must have been invoked with a ``try_apply``
+instruction.  Control will resume in the error destination of
+that instruction.
```

Comment:
Maybe also add that the function is responsible to initialize the indirect error result before `throw_addr`.